### PR TITLE
feat: markuplint-angular-parser supports v2

### PR DESCRIPTION
**Pull request by YusukeHirao**
I heard users hope the angular parser supports.
I support this package because I complete upgrading the basic API of markuplint to v2.

### CODE REVIEW

**Comment by YusukeHirao** [comment]
Hi @JounQin.

Please review this.

File: packages/angular-parser/src/index.ts [hunk 1]
```diff
@@ -244,29 +243,15 @@ const visitor = {
       .replace(/^\[attr\./, '')
       // remove leading `*`, `[]` and `()` wrapper
       .replace(/[()*[\]]/g, '')
-    // remove leading `#`
-    const plainName = potentialName.replace(/#/g, '')
-    const potentialValue = _value.replace(/(^{\s*{)|(}\s*}$)/g, '')
 
     node.potentialName = potentialName
-    node.isInvalid =
```
Comment (JounQin) [inline packages/angular-parser/src/index.ts:252]: Why `isInvalid` property removed?

Comment (YusukeHirao) [reply packages/angular-parser/src/index.ts:252]: @JounQin 

Sorry, I forgot to improve the process on this condition.

I changed the concept of the invalid attribute syntax to throwing an error. The error is caught by the core linter engine then it reports a violation as the parse error. 

https://github.com/markuplint/markuplint/commit/5a0d45e33a460ca6b98349e0e7b6737437150283

**Comment by YusukeHirao** [comment]
@JounQin 

I'm sorry. It didn't pass the tests.

**I think you should revert it.**
I guess fail to design about `isInvalid`.
The `@markuplint/html-parser` doesn't use `isInvalid`.
The `@markuplint/pug-parser` uses `isInvalid` for the parse error.
So the `invalid-attr` rule doesn't use the `isInvalid` property.

I think the rule must use the property and I'm going to change it.

**Comment by JounQin** [comment]
@YusukeHirao Would you like to PR again? I don't have much time working on it recently. 😭

**Comment by YusukeHirao** [comment]
@JounQin Yes, I will do now. Please a moment.

**Pull request merged**